src/components/layer-metadata/layer-metadata.vue

```diff
@@ -61,10 +79,36 @@ function closeLayerMetadata() {
         class="col-span-3 grid gap-2 grid-cols-3"
       >
         <span class="font-bold">{{ t('Description') }}</span>
-        <span
-          v-dompurify-html="layerMetadata.description"
-          class="col-span-2"
-        ></span>
+        <span class="col-span-2">
+          <span v-dompurify-html="description"></span>
+          <button
+            :title="
+              t('Display full description', {
+                ns: 'client',
+              })
+            "
+            v-if="!displayFullDescription"
+            @click="showFullDescription"
+            class="text-secondary hover:underline"
+          >
+            ...
+          </button>
+          <button
+            :title="
+              t('Hide full description', {
+                ns: 'client',
+              })
+            "
+            v-if="
+              displayFullDescription &&
+              (description?.length || '') > MAX_DESCRIPTION_LENGTH
+            "
+            @click="hideFullDescription"
+            class="text-secondary hover:underline"
+          >
+            -
+          </button>
```

Comment:
can this be done using only one button that would handle a toggle function or would it be too wordy?

Reply:
Kept it as is for now, as the content and the title would become conditional.